src/unity/python/setup.py

```diff
@@ -169,6 +172,7 @@ def run(self):
         url='https://github.com/apple/turicreate',
         license='LICENSE.txt',
         description='Turi Create enables developers and data scientists to apply machine learning to build state of the art data products.',
```

Comment:
While we're at it, let's update `description` too. I'm not sure `data products` is the right wording for what we are enabling. Maybe let's take the first sentence or two from the README.

Reply:
Good idea! If we use both sentences, is "in your app" too vague? It might make sense to identify it as particularly useful for deployment on Apple devices.

Reply:
Maybe let's just go with the same short description that we use on the GitHub repo: "Turi Create simplifies the development of custom machine learning models."